test/tools/fuzzTesting/BoostRandomCode.cpp

```diff
@@ -38,9 +38,14 @@ u256 BoostRandomCode::randomUniInt(u256 const& _minVal, u256 const& _maxVal)
 
 void BoostRandomCode::refreshSeed()
 {
-	auto now = std::chrono::steady_clock::now().time_since_epoch();
-	auto timeSinceEpoch = std::chrono::duration_cast<std::chrono::nanoseconds>(now).count();
-	gen.seed(static_cast<unsigned int>(timeSinceEpoch));
+	if (Options::get().randomTestSeed == 0)
+	{
+		auto now = std::chrono::steady_clock::now().time_since_epoch();
+		auto timeSinceEpoch = std::chrono::duration_cast<std::chrono::nanoseconds>(now).count();
+		gen.seed(static_cast<unsigned int>(timeSinceEpoch));
```

Comment:
This should be cast to `gen::result_type`.

Reply:
this page says it has decreasing perfomance 
http://en.cppreference.com/w/cpp/numeric/random/random_device

Reply:
there is no such thing